backend/routes/events.py

```
@@ -0,0 +1,63 @@
+from models.user import User
+from database.session import SessionLocal
+from core.dependencies import get_db
+from api.v1.endpoints.auth import get_current_user
+
+from models.post import Event
+
+router = APIRouter()
+
+@router.get("/events/grouped-by-time", response_model=Dict[str, List[int]])
+def get_grouped_event_ids(db: Session = Depends(get_db)):
+    now = datetime.utcnow()
```

Ruff (0.8.2)

`datetime.datetime.utcnow()` used

(DTZ003)